inst/rmarkdown/templates/flex_dashboard/resources/default.html

```diff
@@ -100,10 +99,24 @@
 
 <script>
 
-$$(document).ready(function () {
+// Navbar padding adjustments
+function adjustForNavHeight() {
+  var navHeight = $$(".navbar").first().outerHeight();
+  var body = $$("body").first().css("padding-top", (navHeight + 8) + "px");
+  var sidebar = $$(".section.sidebar").first().css("top", navHeight + "px");
+}
+
+$$(document).on("shiny:restyle", function() {
```

Comment:
This bit requires https://github.com/rstudio/shiny/pull/3301.

Strictly speaking that bit isn't necessarily, but improves the `bs_themer()` experience with different Bootswatch themes